src/model.py

```diff
@@ -1996,6 +1997,7 @@ def init_components(self, subtasks):
         for stack in range(stack_count):
             for task in self.subtasks:
                 task_layers = self.find(task+"_layers", default=1)
+                # NOTE: if task_layers == 0, empty sequential cnn_0 is made, but data still flows through
```

Comment:
I thought this would throw an error, but feeding a tensor through an empty sequential just results in that tensor at the end. Makes forward pass cleaner